yb-voyager/cmd/analyzeSchema.go

```diff
@@ -619,15 +649,44 @@ func convertIssueInstanceToAnalyzeIssue(issueInstance queryissue.QueryIssue, fil
 		issueType = UNSUPPORTED_PLPGSQL_OBEJCTS
 	}
 
-	//TODO: how to different between same issue on differnt obejct types like ALTER/INDEX for not adding it ot invalid count map
-	increaseInvalidCount, ok := issueInstance.Details["INCREASE_INVALID_COUNT"]
-	if !ok || (increaseInvalidCount.(bool)) {
-		summaryMap[issueInstance.ObjectType].invalidCount[issueInstance.ObjectName] = true
+	var constraintIssues = []string{
+		queryissue.EXCLUSION_CONSTRAINTS,
+		queryissue.DEFERRABLE_CONSTRAINTS,
+		queryissue.PK_UK_ON_COMPLEX_DATATYPE,
+	}
+	/*
+		TODO:
+		// unsupportedIndexIssue
+		// ObjectType = INDEX
+		// ObjectName = idx_name ON table_name
+		// invalidCount.Type = INDEX
+		// invalidCount.Name = ObjectName (because this is fully qualified)
+		// DisplayName = ObjectName
+
+		// deferrableConstraintIssue
+		// ObjectType = TABLE
+		// ObjectName = table_name
+		// invalidCount.Type = TABLE
+		// invalidCount.Name = ObjectName
+		// DisplayName = table_name (constraint_name) (!= ObjectName)
+
+		// Solutions
+		// 1. Define a issue.ObjectDisplayName
+		// 2. Keep it in issue.Details and write logic in UI layer to construct display name.
+	*/
+	displayObjectName := issueInstance.ObjectName
+
+	constraintName, ok := issueInstance.Details[queryissue.CONSTRAINT_NAME]
+	if slices.Contains(constraintIssues, issueInstance.Type) && ok {
+		//In case of constraint issues we add constraint name to the object name as well
+		displayObjectName = fmt.Sprintf("%s, constraint: (%s)", issueInstance.ObjectName, constraintName)
```

Comment:
building displaying name can be a different logic for different issue type.

How about having a function `queryissue.GetDisplayObjectName()` for the time being.
```
func GetDisplayObjectName(issueInstance QueryIssue) string {
}
```

We might need this display name in the assess migration also? How are we dealing it in assess right now?

Reply:
I'm also sort of running into the need for this with regex functions. I want to display the function names used... 

But I think this needs a bit of discussion, there are so many fields out there 😅 it has become confusing. Shall we take this up separately @sanyamsinghal @priyanshi-yb ?

Reply:
Sure. Agree, its becoming a bit confusing.

Reply:
yes, sure